Binary files original/qtconnectivity/src/bluetooth/osx/.osxbtcentralmanager.mm.swp and patch/qtconnectivity/src/bluetooth/osx/.osxbtcentralmanager.mm.swp differ
Binary files original/qtconnectivity/src/bluetooth/osx/.osxbtledeviceinquiry.mm.swp and patch/qtconnectivity/src/bluetooth/osx/.osxbtledeviceinquiry.mm.swp differ
diff -rupN original/qtconnectivity/src/bluetooth/osx/osxbtcentralmanager.mm patch/qtconnectivity/src/bluetooth/osx/osxbtcentralmanager.mm
--- original/qtconnectivity/src/bluetooth/osx/osxbtcentralmanager.mm	2018-03-08 20:26:16.000000000 -0800
+++ patch/qtconnectivity/src/bluetooth/osx/osxbtcentralmanager.mm	2018-03-08 20:28:06.000000000 -0800
@@ -1072,13 +1072,13 @@ QT_USE_NAMESPACE
 {
     using namespace OSXBluetooth;
 
-#if QT_IOS_PLATFORM_SDK_EQUAL_OR_ABOVE(__IPHONE_10_0)
+#if QT_MAC_PLATFORM_SDK_EQUAL_OR_ABOVE(__MAC_10_13, __IPHONE_NA)
     const CBManagerState state = central.state;
 #else
     const CBCentralManagerState state = central.state;
 #endif
 
-#if QT_IOS_PLATFORM_SDK_EQUAL_OR_ABOVE(__IPHONE_10_0)
+#if QT_MAC_PLATFORM_SDK_EQUAL_OR_ABOVE(__MAC_10_13, __IPHONE_NA)
     if (state == CBManagerStateUnknown
         || state == CBManagerStateResetting) {
 #else
@@ -1094,7 +1094,7 @@ QT_USE_NAMESPACE
     }
 
     // Let's check some states we do not like first:
-#if QT_IOS_PLATFORM_SDK_EQUAL_OR_ABOVE(__IPHONE_10_0)
+#if QT_MAC_PLATFORM_SDK_EQUAL_OR_ABOVE(__MAC_10_13, __IPHONE_NA)
     if (state == CBManagerStateUnsupported || state == CBManagerStateUnauthorized) {
 #else
     if (state == CBCentralManagerStateUnsupported || state == CBCentralManagerStateUnauthorized) {
@@ -1114,7 +1114,7 @@ QT_USE_NAMESPACE
         return;
     }
 
-#if QT_IOS_PLATFORM_SDK_EQUAL_OR_ABOVE(__IPHONE_10_0)
+#if QT_MAC_PLATFORM_SDK_EQUAL_OR_ABOVE(__MAC_10_13, __IPHONE_NA)
     if (state == CBManagerStatePoweredOff) {
 #else
     if (state == CBCentralManagerStatePoweredOff) {
@@ -1133,7 +1133,7 @@ QT_USE_NAMESPACE
         return;
     }
 
-#if QT_IOS_PLATFORM_SDK_EQUAL_OR_ABOVE(__IPHONE_10_0)
+#if QT_MAC_PLATFORM_SDK_EQUAL_OR_ABOVE(__MAC_10_13, __IPHONE_NA)
     if (state == CBManagerStatePoweredOn) {
 #else
     if (state == CBCentralManagerStatePoweredOn) {
diff -rupN original/qtconnectivity/src/bluetooth/osx/osxbtcentralmanager_p.h patch/qtconnectivity/src/bluetooth/osx/osxbtcentralmanager_p.h
--- original/qtconnectivity/src/bluetooth/osx/osxbtcentralmanager_p.h	2018-03-08 20:26:16.000000000 -0800
+++ patch/qtconnectivity/src/bluetooth/osx/osxbtcentralmanager_p.h	2018-03-08 20:29:25.000000000 -0800
@@ -58,6 +58,14 @@
 
 #include <Foundation/Foundation.h>
 
+#if QT_MAC_PLATFORM_SDK_EQUAL_OR_ABOVE(__MAC_10_12, __IPHONE_NA)
+#include <CoreBluetooth/CBService.h>
+#include <CoreBluetooth/CBCharacteristic.h>
+#include <CoreBluetooth/CBDescriptor.h>
+#include <CoreBluetooth/CBCentralManager.h>
+#include <CoreBluetooth/CBPeripheral.h>
+#endif
+
 @class QT_MANGLE_NAMESPACE(OSXBTCentralManager);
 
 QT_BEGIN_NAMESPACE
diff -rupN original/qtconnectivity/src/bluetooth/osx/osxbtledeviceinquiry.mm patch/qtconnectivity/src/bluetooth/osx/osxbtledeviceinquiry.mm
--- original/qtconnectivity/src/bluetooth/osx/osxbtledeviceinquiry.mm	2018-03-08 20:26:16.000000000 -0800
+++ patch/qtconnectivity/src/bluetooth/osx/osxbtledeviceinquiry.mm	2018-03-08 20:31:27.000000000 -0800
@@ -40,6 +40,11 @@
 #include <QtCore/qsysinfo.h>
 #include <QtCore/qdebug.h>
 
+#if QT_MAC_PLATFORM_SDK_EQUAL_OR_ABOVE(__MAC_10_12, __IPHONE_NA)
+#import <CoreBluetooth/CBCentralManager.h>
+#import <CoreBluetooth/CBPeripheral.h>
+#endif
+
 QT_BEGIN_NAMESPACE
 
 namespace OSXBluetooth {
@@ -198,7 +203,7 @@ QT_USE_NAMESPACE
     dispatch_queue_t leQueue(qt_LE_queue());
     Q_ASSERT(leQueue);
 
-#if QT_IOS_PLATFORM_SDK_EQUAL_OR_ABOVE(__IPHONE_10_0)
+#if QT_MAC_PLATFORM_SDK_EQUAL_OR_ABOVE(__MAC_10_13, __IPHONE_NA)
     const CBManagerState cbState(central.state);
     if (cbState == CBManagerStatePoweredOn) {
 #else
@@ -219,7 +224,7 @@ QT_USE_NAMESPACE
                                           });
             [manager scanForPeripheralsWithServices:nil options:nil];
         } // Else we ignore.
-#if QT_IOS_PLATFORM_SDK_EQUAL_OR_ABOVE(__IPHONE_10_0)
+#if QT_MAC_PLATFORM_SDK_EQUAL_OR_ABOVE(__MAC_10_13, __IPHONE_NA)
     } else if (cbState == CBManagerStateUnsupported || cbState == CBManagerStateUnauthorized) {
 #else
     } else if (cbState == CBCentralManagerStateUnsupported || cbState == CBCentralManagerStateUnauthorized) {
@@ -233,7 +238,7 @@ QT_USE_NAMESPACE
         }
 
         [manager setDelegate:nil];
-#if QT_IOS_PLATFORM_SDK_EQUAL_OR_ABOVE(__IPHONE_10_0)
+#if QT_MAC_PLATFORM_SDK_EQUAL_OR_ABOVE(__MAC_10_13, __IPHONE_NA)
     } else if (cbState == CBManagerStatePoweredOff) {
 #else
     } else if (cbState == CBCentralManagerStatePoweredOff) {
